docs/documentation.rst

```diff
@@ -318,67 +318,37 @@ opening the file ``qc/index.html``. Use the "Search"
+  display vertebral labeling
+- Review all spinal cord and gray matter segmentations and vertebral labeling. Use the keyboard shortcuts up/down arrow
+  to switch between subjects and the left arrow to toggle overlay.
+- If you spot *major* issues with the segmentation (e.g. noticeable leaking or under-segmentation that extends over
+  several slices) or wrong labeling, add the image name into the yaml file as in an example below:
 
-.. code-block:: bash
+.. Hint::
+    You can create yaml file easily using your text editor (vim, nano, atom, ...).
+
+::
+
+    FILES_SEG:
+    - sub-amu01_T1w_RPI_r.nii.gz
+    - sub-amu01_T2w_RPI_r.nii.gz
+    - sub-cardiff02_dwi_moco_dwi_mean.nii.gz
+    FILES_GMSEG:
+    - sub-amu01_T2star_rms.nii.gz
+    FILES_LABEL:
+    - sub-amu01
+    - sub-amu02
 
-  #!/bin/bash
-  # Folder to output the manual labels
-  PATH_SEGMANUAL="../../seg_manual"
-  mkdir $PATH_SEGMANUAL
-  # List of files to correct segmentation on
-  FILES=(
-  sub-amu02_acq-T1w_MTS.nii.gz
-  sub-beijingGE04_T2w_RPI_r.nii.gz
-  sub-brnoPrisma01_T2star_rms.nii.gz
-  sub-geneva04_dwi_crop_moco_dwi_mean.nii.gz
-  )
-  # Loop across files
-  for file in ${FILES[@]}; do
-    # extract subject using first delimiter '_'
-    subject=${file%%_*}
-    # check if file is under dwi/ or anat/ folder and get fname_data
-    if [[ $file == *"dwi"* ]]; then
-      fname_data=$subject/dwi/$file
-    else
-      fname_data=$subject/anat/$file
-    fi
-    # get fname_seg depending if it is cord or GM seg
-    if [[ $file == *"T2star"* ]]; then
-      fname_seg=${fname_data%%".nii.gz"*}_gmseg.nii.gz${fname_data##*".nii.gz"}
-      fname_seg_dest=${PATH_SEGMANUAL}/${file%%".nii.gz"*}_gmseg-manual.nii.gz${file##*".nii.gz"}
-    else
-      fname_seg=${fname_data%%".nii.gz"*}_seg.nii.gz${fname_data##*".nii.gz"}
-      fname_seg_dest=${PATH_SEGMANUAL}/${file%%".nii.gz"*}_seg-manual.nii.gz${file##*".nii.gz"}
-    fi
-    # Copy file to PATH_SEGMANUAL
-    cp $fname_seg $fname_seg_dest
-    # Launch FSLeyes
-    echo "In FSLeyes, click on 'Edit mode', correct the segmentation, then save it with the same name (overwrite)."
-    fsleyes -yh $fname_data $fname_seg_dest -cm red
-  done
-
-- Make this script executable:
-
-  .. code-block:: bash
-
-     chmod 775 fix_seg.sh
-
-- In the QC report, enter the string "deepseg" to only display segmentation results.
-- Review all segmentations. Use the keyboard shortcuts up/down arrow to switch between
-  subjects and the left arrow to toggle overlay.
-- If you spot *major* issues with the segmentation (e.g. noticeable leaking or under-segmentation that extends over several slices),
-  add the image name in the variable array ``FILES`` in the script.
-- If the data quality is too low to be interpreted (too blurry, large artifacts),
-  add the image file name to the variable ``TO_EXCLUDE`` in the file ``parameters.sh``,
-  which will be used in the next processing iteration.
```

Comment:
This point was out-of-date because `generate_figure.py` no longer requires `parameters.sh` file. Subjects to remove and sites to exclude are defined directly inside `generate_figure.py` script. Is it okay for future? Because now, user has to modify the script. Or do we plan to create yaml file containing this problematic sites/subjects/segmentations and pass this yaml file as input to this script?

Reply:
Yes, yaml would be nice indeed. Can you pls open an issue.

A subtlety: it is possible to exclude subject during processing by passing -exclude-list in sct run batch 

For now you can update the text to conform to the current behaviour